server/game/cards/02_SHD/units/DiscerningVeteran.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer, ZoneName } from '../../../core/Constants';
+
+export default class DiscerningVeteran extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '9765804063',
+            internalName: 'discerning-veteran',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'This unit captures an enemy non-leader ground unit',
+            targetResolver: {
+                controller: RelativePlayer.Opponent,
+                zoneFilter: ZoneName.GroundArena,
+                immediateEffect: AbilityHelper.immediateEffects.capture()
+            }
```

Comment:
Do you also need to filter out leader units or it's the default?

Reply:
Yeah it's the default since CaptureSystem declares its target type to NonLeaderUnit